configure/os/CONFIG.Common.RTEMS-mvme2700

```diff
@@ -1,7 +1,11 @@
 #
 # Author: Matt Rippa
 #
+ifeq ($(shell test $(RTEMS_VERSION) -gt 5; echo $$?),0)
```

Comment:
Is this expecting an environment variable `RTEMS_VERSION`? Currently `RTEMS_VERSION` is set by the user from `$(CONFIG)/os/CONFIG_SITE.Common.RTEMS`, which is included by `$(CONFIG)/os/CONFIG.Common.RTEMS`.

To use the `RTEMS_VERSION` from the config files instead of an environment variable, can we put this check after the config include? Moreover, the shell `test` command will probably not be compatible with Windows.

I propose a small change to this commit here: https://github.com/kiwichris/epics-base/pull/1/commits/acde5b647654d13fa7dea5cf42a1120cf5760a1d

My proposed PR uses only make functions and puts the check at the bottom to enable setting the version from make config files instead of build environment. Can we use something like this?

Reply:
Your change is valid. I have not merged it into this patch set but I can if you are OK with this?

Reply:
Thanks!
Yes, I am OK with it being merged into this patch set.